DemoApp-Swift/DemoApp-Swift/Ads/ALAutoLayoutBannerAdViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ALAutoLayoutBannerAdViewController.swift
+//  DemoApp-Swift
+//
+//  Created by Andrew Tian on 9/20/19.
+//  Copyright © 2019 AppLovin. All rights reserved.
+//
+
+import UIKit
+import AppLovinSDK
+
+class ALAutoLayoutBannerAdViewController: UIViewController, MAAdViewAdDelegate
+{
+    private var adView: MAAdView?
+    
+    // MARK: View Lifecycle
+    
+    override func viewDidLoad()
+    {
+        super.viewDidLoad()
+        
+        adView = MAAdView(adUnitIdentifier: "BANNER_AD_UNIT_ID")
+        guard let adView = adView else { return }
```

Comment:
Can you explain how this can be possible? why do we need to check input on something we know exists? Isn't that bloated code and misdirection?